src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        // create three balls using class Lottery and print information about them in
+        // console
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
```

Comment:
Make all magic number constant

Reply:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants